src/components/Button/Button.tsx

```diff
@@ -58,17 +63,24 @@ const Button: FC<Props> = ({
   isDisabled = false,
   fullWidth,
   onClick,
+  size = 'md',
   tabIndex,
   isLoading,
   onFocus,
   onBlur,
 }) => {
   const disabled = isLoading || isDisabled;
 
-  const buttonClasses = classNames('Palmetto-Button', className, {
-    loading: isLoading,
-    fullWidth,
-  });
+  const buttonClasses = classNames(
```

Comment:
I prefer automatically wrapping arguments to classNames to the next line, as is done here. Is this something we should standardize?